pkg/builtinchecks/yamls/improper-container-image-tag.yaml

```diff
@@ -0,0 +1,9 @@
+name: "improper-container-image-tag"
```

Comment:
We should just call this `latest-tag`, yeah?

Reply:
Thank you. Yes, I renamed it to "latest-tag"